pkg/controllers/manifestwork.go

```diff
@@ -245,12 +245,12 @@ func (r *HypershiftDeploymentReconciler) createOrUpdateMainfestwork(ctx context.
 
 	passedSecurity, statusUpdateErr := r.validateSecurityConstraints(ctx, hyd)
 	if !passedSecurity {
-		return ctrl.Result{}, statusUpdateErr
+		return ctrl.Result{RequeueAfter: time.Minute * 1}, statusUpdateErr
 	}
 
 	m, err := scaffoldManifestwork(hyd)
 	if err != nil {
-		return ctrl.Result{RequeueAfter: time.Minute * 1}, err
+		return ctrl.Result{}, err
```

Comment:
why not requeue here?

Reply:
When err is not nil so it automatically re-queue. This is how it was before my previous PR. I placed the minute reconcile in the wrong return.